Use backticks to render URLs etc as monospaced

peterbrightwell opened the pull request:
Currently these use 'quotes' for URLs, parameter names etc, but more recent NMOS specs have used  backticks  for monospace rendering., This is consistent with how examples and other codeblocks appear and look better.

This should be merged (if we choose to do it) before merging docs-language.

peterbrightwell committed:
Backticks for 1.0 ... 3.1

peterbrightwell committed:
Some more backticks

peterbrightwell committed:
Some more backticks

peterbrightwell committed:
Backtick HTTP return codes

garethsb commented on docs/1.0. Overview.md:
```diff
@@ -4,7 +4,7 @@ _(c) AMWA 2017, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 ## Introduction
 
-AMWA IS-05 specifies how to allow a Device in an NMOS compatible system to connect to other Devices, by means of the Devices' Senders and Receivers.
+AMWA IS-05 specifies how to allow a Device in an NMOS compatible system to connect to other Devices, by means of the Devices` Senders and Receivers.
```
That's an apostrophe...

```suggestion
AMWA IS-05 specifies how to allow a Device in an NMOS compatible system to connect to other Devices, by means of the Devices' Senders and Receivers.
```

garethsb commented on docs/2.0. APIs.md:
```diff
@@ -15,11 +15,11 @@ Examples of JSON format output are provided in the [examples](../examples/) fold
 ### Content Types
 
-All APIs MUST provide a JSON representation signalled via 'Content-Type: application/json' headers. This SHOULD be the default content type in the absence of any requested alternative by clients. Other content types (such as HTML) are permitted if they are explicitly requested via Accept headers.
+All APIs MUST provide a JSON representation signalled via `Content-Type: application/json` headers. This SHOULD be the default content type in the absence of any requested alternative by clients. Other content types (such as HTML) are permitted if they are explicitly requested via Accept headers.
```
```suggestion
All APIs MUST provide a JSON representation signalled via `Content-Type: application/json` headers. This SHOULD be the default content type in the absence of any requested alternative by clients. Other content types (such as HTML) are permitted if they are explicitly requested via `Accept` headers.
```

garethsb commented on docs/2.0. APIs.md:
````diff
@@ -106,6 +106,6 @@ For HTTP codes 400 and upwards, a JSON format response MUST be returned as follo
 ```
 
-In the above example, the 'code' SHOULD match the HTTP status code. 'error' MUST always be present and in string format. 'debug' MAY be null if no further debug information is available.
+In the above example, `code` SHOULD match the HTTP status code. `error` MUST always be present and in string format. `debug` MAY be null if no further debug information is available.
````
```suggestion
In the above example, `code` SHOULD match the HTTP status code. `error` MUST always be present and in string format. `debug` MAY be `null` if no further debug information is available.
```

garethsb commented on docs/2.2. APIs - Server Side Implementation.md:
```diff
@@ -17,18 +17,18 @@ Access-Control-Allow-Headers: Content-Type, Accept
 
 - Senders and Receivers SHOULD present sensible default values on transport parameter endpoints. Suggested defaults are given in the relevant schemas. Any default parameters such as port numbers specified in relevant RFCs SHOULD be followed. Parameters where no sensible defaults exist, such as source IP address on a Receiver, SHOULD be set to null.
-- Senders that are not configured (for example have a null source IP address value) SHOULD return 404 on their active transport file endpoint, until a usable set of parameters has been activated.
+- Senders that are not configured (for example have a null source IP address value) SHOULD return `404` (Not Found) on their active transport file endpoint, until a usable set of parameters has been activated.
```
```suggestion
- Senders that are not configured (for example have a `null` source IP address value) SHOULD return `404` (Not Found) on their active transport file endpoint, until a usable set of parameters has been activated.
```

garethsb commented on docs/2.2. APIs - Server Side Implementation.md:
```diff
@@ -17,18 +17,18 @@ Access-Control-Allow-Headers: Content-Type, Accept
 
 When a Sender or Receiver is first started it might not have all the parameters it needs to operate. For example IP addresses and port numbers might not be set. In such cases:
 
 - Senders and Receivers SHOULD present sensible default values on transport parameter endpoints. Suggested defaults are given in the relevant schemas. Any default parameters such as port numbers specified in relevant RFCs SHOULD be followed. Parameters where no sensible defaults exist, such as source IP address on a Receiver, SHOULD be set to null.
```
```suggestion
- Senders and Receivers SHOULD present sensible default values on transport parameter endpoints. Suggested defaults are given in the relevant schemas. Any default parameters such as port numbers specified in relevant RFCs SHOULD be followed. Parameters where no sensible defaults exist, such as source IP address on a Receiver, SHOULD be set to `null`.
```

garethsb commented on docs/3.1. Interoperability - NMOS IS-04.md:
```diff
@@ -8,11 +8,11 @@ When this API is used alongside IS-04 in a deployment, the IS-04 APIs SHOULD be
-The Connection Management API SHOULD be advertised as a 'control' endpoint when publishing a compliant NMOS Device.
-Control interfaces can identify all Devices which implement the Connection Management API by a 'type' Uniform Resource Name (URN) having the base 'urn:x-nmos:control:sr-ctrl' followed by a '/' character and the API version.
-The associated 'href' is the URL of the Connection API base resource.
+The Connection Management API SHOULD be advertised as a `control` endpoint when publishing a compliant NMOS Device.
```
```suggestion
The Connection Management API SHOULD be advertised as a control endpoint when publishing a compliant NMOS Device.
```

garethsb commented on docs/3.1. Interoperability - NMOS IS-04.md:
```diff
@@ -25,29 +25,29 @@ The associated 'href' is the URL of the Connection API base resource.
 More details about multi-version support can be found in [Upgrade Path](5.0.%20Upgrade%20Path.md).
 
-A given instance of the Connection Management API MAY offer control of multiple Devices in a Node from a single URI. Alternatively there MAY be multiple instances of the API on one Node, each corresponding to one Device. A separate 'control' endpoint for each Device's Connection Management instance MUST be advertised, even if the URI is the same.
+A given instance of the Connection Management API MAY offer control of multiple Devices in a Node from a single URI. Alternatively there MAY be multiple instances of the API on one Node, each corresponding to one Device. A separate `control` endpoint for each Device's Connection Management instance MUST be advertised, even if the URI is the same.
```
```suggestion
A given instance of the Connection Management API MAY offer control of multiple Devices in a Node from a single URI. Alternatively there MAY be multiple instances of the API on one Node, each corresponding to one Device. A separate endpoint for each Device's Connection Management instance MUST be advertised in the Device's `controls`, even if the URI is the same.
```

garethsb commented on docs/3.1. Interoperability - NMOS IS-04.md:
```diff
@@ -25,29 +25,29 @@ The associated 'href' is the URL of the Connection API base resource.
 
 ## Identifying Active Connections
 
 In order to populate the `subscription` attribute of IS-04 Senders and Receivers, the Connection Management API includes keys for `sender_id` and `receiver_id` in its staged parameters. These SHOULD be used to signal that a Sender or Receiver is being connected to another NMOS compatible Sender or Receiver. It is the client's responsibility to set and unset (using null) the `sender_id` or `receiv ...
```
```suggestion
In order to populate the `subscription` attribute of IS-04 Senders and Receivers, the Connection Management API includes keys for `sender_id` and `receiver_id` in its staged parameters. These SHOULD be used to signal that a Sender or Receiver is being connected to another NMOS compatible Sender or Receiver. It is the client's responsibility to set and unset (using `null`) the `sender_id` or `receiver_id` parameters when modifying the `transport_params` or `transport_file`.
```

garethsb reviewed (changes requested):
First pass

garethsb committed:
Apply suggestions from code review

garethsb committed:
Sync common text with IS-04 docs-language PR.
STRONGLY isn't a conformance language key word.
Endpoints are styled with a preceding backslash.

garethsb committed:
Backtick "auto" (leaving the quotes, as it's a string value not an attribute name or a value keyword like null, true or false)

garethsb reviewed (approved):
LGTM
